h2o-extensions/target-encoder/src/main/java/hex/schemas/TargetEncoderV3.java

```diff
@@ -21,6 +21,9 @@
 
     @API(help = "Data leakage handling strategy. Default to None.", values = {"None", "KFold", "LeaveOneOut"})
     public TargetEncoder.DataLeakageHandlingStrategy data_leakage_handling;
+
+    @API(help = "Noise level", required = false, direction = API.Direction.INPUT, gridable = true)
+    public double noise_level;
```

Comment:
Is it a percentage or sth? The doc string may need more explanations.

Reply:
OK, checked impl. details: no dimension.

Reply:
Still, this `help` is used as field documentation in Flow, would be nice to provide a recommended range for example.